client/my-sites/sidebar/sidebar.jsx

```diff
@@ -376,35 +371,28 @@ export class MySitesSidebar extends Component {
 			return null;
 		}
 
-		const storeLink = '/store' + siteSuffix;
-
-		const isJetpackOrAtomicSite =
-			isJetpack && canUserManageOptions && this.props.isSiteAutomatedTransfer;
+		const isPermittedSite = isJetpack && canUserManageOptions && this.props.isSiteAutomatedTransfer;
```

Comment:
I realize this code is the same as before almost but isn't a site `isJetpack` always true when `isSiteAutomatedTransfer`?

Reply:
I'm not sure but I think you're right.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/21466/commits/7912b018ccbea3295bfe1fa768749c4b2286495a